lib/main.js

```diff
@@ -45,8 +45,21 @@ class PythonLanguageClient extends AutoLanguageClient {
     return "ide-python"
   }
 
+  getPyLs() {
+    if (this.pyls === undefined) {
+      let pyls = atom.config.get("ide-python.pyls") || "pylsp"
+      // check if it exists
+      if (whichSync(pyls, { nothrow: true }) === null) {
+        pyls = "pyls"
```

Comment:
This should be like `pyls = (pyls === "pylsp") ? "pyls" : "pylsp"` so if  `ide-python.pyls` is not the default it falls back to `pylsp` instead of `pyls`. 
Maybe check further for pylsp and fallback to pyls : ide-python.pyls --> pylsp --> pyls

Reply:
```suggestion
		// use the other one if it doesn't exist
        pyls = (pyls === "pylsp") ? "pyls" : "pylsp"
```